tigerpath/majors_and_certificates/majors/HIS.json

```diff
@@ -95,285 +100,606 @@
+                        "CLA 216/HIS 216: Archaic and Classical Greece",
+                        "CLA 217/HIS 217: The Greek World in the Hellenistic Age",
+                        "CLA 218/HIS 218: The Roman Republic",
+                        "CLA 219/HIS 219: The Roman Empire, 31 B.C. to A.D. 337",
+                        "NES 220/HIS 220: Jews, Muslims, and Christians in the Middle Ages",
+                        "NES 390/HIS 221: Medieval Cairo: A Survival Guide",
+                        "MED 227/HIS 227: The Worlds of the Middle Ages",
+                        "CLA 231/HIS 231: Ancient Greek and Roman Medicine: Bodies, Physicians, and Patients",
+                        "JDS 232/HIS 232: Hatred or Tolerance? Jews & Christians in Medieval and Early Modern Europe",
+                        "NES 369/HIS 251: The World of the Cairo Geniza",
+                        "EAS 280/HIS 279: Nomadic Empires",
+                        "HIS 294: What Is the Scientific Revolution?",
+                        "HIS 303: Colonial Latin America to 1810",
+                        "EAS 409/HIS 309: The Warrior Culture of Japan",
+                        "HIS 314: Precolonial Africa",
+                        "HIS 324 : Early Modern China",
+                        "CLA 326/HIS 326: Topics in Ancient History - Town and Country",
+                        "CLA 327/HIS 327: Topics in Ancient History",
+                        "CLA 324/HIS 328: Classical Historians and Their Philosophies of History",
+                        "HIS 332: India Before Europe",
+                        "EAS 317/HIS 335: The Warrior Culture of Japan",
+                        "NES 437/HIS 337: The Ottoman Empire, 1300-1800",
+                        "HIS 343: The Formation of Europe in the First Millennium",
+                        "HIS 344: The Civilization of the High Middle Ages",
+                        "HIS 345: The Crusades",
+                        "HUM 320/HIS 346: Making Medieval Worlds: Methods and Materials",
+                        "HIS 367: English Constitutional History",
+                        "HIS 371: The Colonization of North America",
+                        "HIS 413: Medieval Democracy: Italian City States of the Middle Ages",
+                        "HIS 421: Venice and the Mediterranean World",
+                        "HIS 422: Hindu, Muslim, Untouchable",
+                        "HIS 423: The History of Christianity in Africa: From St. Mark to Desmond Tutu",
+                        "HIS 437: Law after Rome",
+                        "HIS 452: Magic, Matter, Medicine: Science in the Medieval World",
+                        "HIS 491: Fertile Bodies: A Cultural History of Reproduction from Antiquity to the Enlightenment",
+                        "HIS 495: Alchemy: Art and Science"
+                      ]
+                    }
+                  ]
+                },
+                { 
+                  "name":"Geographical Distribution Requirement",

```

Comment:
https://history.princeton.edu/academics/undergraduate/current-concentrators#DistributionReqs

"Geographical Distribution Requirement
Students in the class of 2023 are required to take at least two (2) courses that are principally focused on Africa, Asia, Latin America, or the Middle East. These courses may double count with the thematic areas."